mango_base/mango/dashboards/time_series_app.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez